airflow/sentry.py

```diff
@@ -78,14 +79,15 @@ class ConfiguredSentry(DummySentry):
         def __init__(self):
             """Initialize the Sentry SDK."""
             ignore_logger("airflow.task")
-            executor_name = conf.get("core", "EXECUTOR")
 
             sentry_flask = FlaskIntegration()
 
             # LoggingIntegration is set by default.
             integrations = [sentry_flask]
 
-            if executor_name == "CeleryExecutor":
+            executor_class, _ = ExecutorLoader.import_executor_cls(conf.get("core", "EXECUTOR"))
```

Comment:
You can use the helper method I added for this `ExecutorLoader.import_default_executor_cls()`